app/services/cocina_generator/description/generator.rb

```diff
@@ -134,7 +134,7 @@ def date_props_for(date, type: nil)
         {
           type: type,
           qualifier: date.uncertain? ? 'approximate' : nil,
-          value: date.uncertain? ? date.to_s : date.edtf
+          value: date.edtf.chomp('?')
         }.compact
```

Comment:
edtf formatted dates include a `?` when approximate, just chomp it anytime it is there